docs/getting_started.rst

```diff
@@ -130,13 +130,13 @@ Some of the intricacies from using the compiler are documented in the `Compiler
 The debug build output
 ======================
 
-The ``index-template.html`` and its corresponding ``index.js`` file define how the main page is packaged up by opensphere-build-index_. That script produces ``index.html``, which is the debug instance. It contains all of the vendor scripts and css in addition to all of the source files listed from the `Closure Compiler`_ manifest (``.build/gcc-manifest``).
+The ``index-template.html`` and its corresponding ``index.js`` file define how the main page is packaged up by opensphere-build-index_. That script produces ``index.html``, which is the debug instance. It contains all of the vendor scripts and css in addition to source files that load the application using Closure's debug loader.
 
 .. _opensphere-build-index: https://github.com/ngageoint/opensphere-build-index
 
 If you set up nginx or httpd as recommended above, accessing it might be accomplished by pointing your browser at http://localhost:8080/workspace/opensphere
 
-Note: because the debug instance references each individual Javascript file in place, it can result in the debug page referencing thousands of individual files. The only browser that handles this gracefully (as of this writing) is Chrome. Firefox technically works but is much more painful.
+.. note:: The debug instance references each individual Javascript file in place, resulting in the debug page loading thousands of individual files. Modern browsers tend to handle this well, but debugging on older browsers can be a painful experience.
```

Comment:
Its not literally "painful", so maybe "slow, and much less productive," instead. Just thinking of non-native speakers.

Reply:
Pain isn't always physical, and pre-Quantum Firefox debugging has caused a fair amount of mental anguish. 😉 Regardless, I adjusted the syntax a bit.